django/conf/__init__.py

```diff
@@ -39,6 +39,18 @@
     "it will be removed in Django 5.0."
 )
 
+# RemovedInDjango51Warning
+ASSERT_FORM_SET_ERROR_RENAMED_MSG = (
+    "assertFormsetError has been renamed to assertFormSetError. Support for "
+    "the old spelling will be removed in Django 5.1."
+)
+
+# RemovedInDjango51Warning
+ASSERT_QUERY_SET_EQUAL_RENAMED_MSG = (
+    "assertQuerysetEqual has been renamed to assertQuerySetEqual. Support for "
+    "the old spelling will be removed in Django 5.1."
+)
+
```

Comment:
`django/conf/__init__.py` is not a proper place for this messages as `assertQuerysetEqual()`/`assertFormsetError()` are test assertions not settings. I would put them directly in the `warnings.warn(...)`.

Reply:
Fixed in c990ec5dc5860893ecccb397aba4de544149ffb7.